clisops/core/subset.py

```diff
@@ -58,8 +62,19 @@ def func_checker(*args, **kwargs):
                 stacklevel=2,
             )
             kwargs["start_date"] = da.time.min().dt.strftime("%Y").values
+        except ValueError:
+            warnings.warn(
+                '"start_date" has been nudged to nearest valid time step in xarray object.',
+                UserWarning,
+                stacklevel=2,
+            )
+            nudged = da.time.sel(time=slice(kwargs["start_date"], None)).values[0]
```

Comment:
I suggest to use nearest neighbour instead: http://xarray.pydata.org/en/stable/indexing.html#nearest-neighbor-lookups

Reply:
@huard, given that we are asking for a range, the nearest neighbour lookup might include values outside of the requested range. The proposed solution will look _up_ from the lower bound and _down_ from the upper bound.